sky/cli.py

```diff
@@ -3231,9 +3231,6 @@ def show_gpus(
     type is the lowest across all regions for both on-demand and spot
     instances. There may be multiple regions with the same lowest price.
     """
-    # validation for the --cloud kubernetes
-    if cloud == 'kubernetes':
-        raise click.UsageError('Kubernetes does not have a service catalog.')
```

Comment:
If I have a Kubernetes cluster without any GPUs, I get an empty output:
```
(base) ➜  ~ sky show-gpus --cloud kubernetes


Hint: use -a/--all to see all accelerators (including non-common ones) and pricing.
```

On following the hint and running `sky show-gpus --cloud kubernetes -a`:
```
$ sky show-gpus --cloud kubernetes -a

Resources 'None' not found. Try 'sky show-gpus --all' to show available accelerators.
```

If `cloud==kubernetes` and the cluster doesn't contain any GPUs, can we return a message from cli.py saying:
```
"No GPUs found in Kubernetes cluster. If your cluster contains GPUs, make sure nvidia.com/gpu resource is available on the nodes and the node labels for identifying GPUs (e.g., skypilot.co/accelerators) are setup correctly. To further debug, run: sky check."
```

Reply:
Sounds good. I will add the error message in case of no GPUs.

Reply:
Done in https://github.com/skypilot-org/skypilot/pull/2638/commits/fa0df6ac83a2cdb8b50e6e6c78f472d3e2d70a4f

Reply:
It now shows
```
sky show-gpus --cloud kubernetes 
No GPUs found in Kubernetes cluster. If your cluster contains GPUs, make sure nvidia.com/gpu resource is available on the nodes and the node labels for identifying GPUs (e.g., skypilot.co/accelerators) are setup correctly. To further debug, run: sky check.
```